### ISSUE TITLE
add Ubuntu support

### ISSUE THREAD

**Issue by jakovm** [prestodb/ambari-presto-service#18]
Hello, 
thank you for your great work!
Is it possilbe to add Ubuntu support?

**Comment by petroav** [comment]
Given that Presto runs on Ubuntu it shouldn't be too difficult to add that support. However, I'm afraid that work isn't on our current roadmap.

**Comment by mattsfuller** [comment]
Hi @jakovm, thank you! As Anton said, we don't have any near term plans for it. But we welcome contributions if it's something you want to work on. We'd be happy to provide guidance.

**Comment by jakovm** [comment]
Hi @mattsfuller, I would be happy to help here. Since I'm pretty new to programming I would need guidance. So what I need to do add ubuntu support?

**Comment by petroav** [comment]
@jakovm -- if you're new to programming then this may not be a good place for you to start. Even though the programming needed to complete this task is trivial, figuring out how all the pieces fit together is hard. 

If you'd still like to attempt the task then you'll need to edit the scripts in `package/scripts`. Specifically you'll have to change (1) the way we install Presto and (2) the way we configure Presto.

(1) should just be a matter of downloading the `tar.gz` distribution of Presto and unpacking it. You'll have to somehow detect whether you're running on Ubuntu and switch to the `tar.gz` based on that. The installation happens in the `install()` method for both the coordinator and workers.
(2) is a little more complicated because the configuration files will now have different paths because we didn't install Presto using the `rpm`.

Finally, you should test your changes on an Ubuntu cluster and write some unit tests. For unit testing I had to do some hacking because the `resource_management` package from Ambari is not available as standalone. To get around that I created a dummy package in the same directory as my tests, modified the `PYTHONPATH` and then mocked all the objects I needed to mock.

Before you start coding you should familiarize yourself with Ambari's architecture by watching the two Ambari webinars [here](http://hortonworks.com/partners/learn/#ambari). The Ambari Operations webinar is not very relevant to this task but will give you an introduction on some of the features. The Ambari Stacks and Views webinar is very relevant. For examples on other integration code you can check out these [Ambari workshops](https://github.com/abajwa-hw/ambari-workshops).

**Comment by jakovm** [comment]
Hi @petroav thanks for the instructions. Will try to get it done, but will probably take some time...

**Comment by mrveera** [comment]
Hey @petroav , @jakovm 
Any updates on this card. @jakovm Are you still working on this??

**Comment by mattsfuller** [comment]
I don't believe anyone is working on this at the moment. We'd love to have other contribute.

**Comment by mrveera** [comment]
I can work on it but I need some context about installing and adding configuration
